src/components/ViewTask.js

```diff
@@ -0,0 +1,55 @@
+/** @jsx jsx */
+import { jsx } from 'theme-ui';
+import PropTypes from 'prop-types';
+import { AiTwotoneCalendar } from 'react-icons/ai';
+
+function ViewTask({ task }) {
+  const h3Div = {
+    display: 'flex',
+  };
+  const AiTwoTone = {
+    paddingTop: '25px',
+  };
+  const dueDate = {
+    paddingLeft: '20px',
+    color: '#b6b6b6',
+  };
+  const dueDateActual = {
+    paddingLeft: '40px',
+    marginTop: '5px',
+  };
+  const categoryName = {
+    paddingLeft: '35px',
+  };
+
+  return (
+    <div>
+      <h1>{task.task_name}</h1>
+      <div style={h3Div}>
```

Comment:
Don't forget to use `sx` instead of `style` ;)

Reply:
Will do. Thanks for the reminder.